containers/jetty/src/main/java/com/yahoo/athenz/container/AthenzJettyContainer.java

```diff
@@ -389,7 +415,15 @@ void loadServicePrivateKey() {
             sslContextFactory.setWantClientAuth(true);
         }
 
-        sslContextFactory.setEnableOCSP(enableOCSP);
+        if (enableOCSP) {
+            // See https://stackoverflow.com/questions/49904935/jetty-9-enable-ocsp-stapling-for-domain-validated-certificate
+            Security.setProperty("ocsp.enable", "true");
+            System.setProperty("jdk.tls.server.enableStatusRequestExtension", "true");
+            System.setProperty("com.sun.net.ssl.checkRevocation", "true");
+            sslContextFactory.setEnableOCSP(true);
+            sslContextFactory.setValidateCerts(true);   // not sure what this does... OCSP works even without this - but it is recommended.
```

Comment:
other than sslContextFactory.setEnableOCSP(true); I'm not sure why we have the other settings or why we're setting them in the first place. So let's remove those from our code. if the organization wants to enable those, they can do so by adding them to the server startup script as just regular system properties.